bot/exts/evergreen/issues.py

```diff
@@ -86,70 +109,85 @@ def check_in_block(message: discord.Message, repo_issue: str) -> bool:
+            else:
+                icon_url = Emojis.issue_closed
+
+        # If the 'issues' key is not contained in the API response and there is no error code, then
+        # we know that a PR has been requested and a call to the pulls API endpoint is necessary
+        # to get the desired information for the PR.
+        else:
+            log.trace(f"PR provided, querying GH pulls API for additional information: {merge_url}")
+            async with self.bot.http_session.get(merge_url) as m:
                 if json_data.get("state") == "open":
-                    icon_url = Emojis.issue
+                    icon_url = Emojis.pull_request
+                # When the status is 204 this means that the state of the PR is merged
+                elif m.status == 204:
+                    icon_url = Emojis.merge
                 else:
-                    icon_url = Emojis.issue_closed
-
-            # If the 'issues' key is not contained in the API response and there is no error code, then
-            # we know that a PR has been requested and a call to the pulls API endpoint is necessary
-            # to get the desired information for the PR.
-            else:
-                log.trace(f"PR provided, querying GH pulls API for additional information: {merge_url}")
-                async with self.bot.http_session.get(merge_url) as m:
-                    if json_data.get("state") == "open":
-                        icon_url = Emojis.pull_request
-                    # When the status is 204 this means that the state of the PR is merged
-                    elif m.status == 204:
-                        icon_url = Emojis.merge
-                    else:
-                        icon_url = Emojis.pull_request_closed
+                    icon_url = Emojis.pull_request_closed
 
-            issue_url = json_data.get("html_url")
-            links.append([icon_url, f"[{repository}] #{number} {json_data.get('title')}", issue_url])
+        issue_url = json_data.get("html_url")
 
-        return links
+        return IssueState(repository, number, issue_url, json_data.get('title', ''), icon_url)
 
     @staticmethod
-    def get_embed(result: list, user: str = "python-discord", repository: str = "") -> discord.Embed:
-        """Get Response Embed."""
-        description_list = ["{0} [{1}]({2})".format(*link) for link in result]
+    def format_embed(
+            results: t.List[t.Union[IssueState, FetchError]],
+            user: str,
+            repository: t.Optional[str] = None
+    ) -> discord.Embed:
+        """Take a list of IssueState or FetchError and format a Discord embed for them."""
+        description_list = []
+
+        for result in results:
+            if isinstance(result, IssueState):
+                description_list.append(f"{result.icon_url} [{result.title}]({result.url})")
+            elif isinstance(result, FetchError):
+                description_list.append(f"[{result.return_code}] {result.message}")
```

Comment:
You can add a :x: emoji, ahead of the return code to match the pattern.
```suggestion
                description_list.append(f":x: [{result.return_code}] {result.message}")
```